internal/pkg/propagation/summary/testdata/src/summary_analysistest/tests/test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
```

Comment:
As a reminder, the `//want <string>` match as regexp.  You have the `\Q` in the one test case, but that might be useful throughout, even though I don't think we are at any real risk of matching `encoding/json2Unmarshal` or something.

However, we probably _should_ anchor the test strings with `^...$`.  This test suggests we are getting exactly `fmt.Println`, and not, for instance, `fmt.Println()`.

Reply:
Good point! Done.